test/utils/review.ts

```diff
@@ -1,42 +1,51 @@
-import nock from 'nock'
-import { composeReviewDismissalMsg } from '../../src/msg_composer'
+import nock from "nock";
+import { composeReviewDismissalMsg } from "../../src/msg_composer";
 
-export function checkApproved(done: jest.DoneCallback, pullNumber: Number = 1): void {
-  nock('https://api.github.com')
-    .post(`/repos/tianhaoz95/approveman-test/pulls/${pullNumber}/reviews`, (body: any) => {
-      done(expect(body).toMatchObject({
-        event: 'APPROVE'
-      }))
-      return true
-    })
-    .reply(200)
-}
+export const checkApproved = (pullNumber: Number = 1): void => {
+  nock("https://api.github.com")
+    .post(
+      `/repos/tianhaoz95/approveman-test/pulls/${pullNumber}/reviews`,
+      (body: any) => {
+        expect(body).toMatchObject({
+          event: "APPROVE",
+        });
+        return true;
+      },
+    )
+    .reply(200);
+};
 
-export function setPreviousReviews(reviews: any[]): void {
-  nock('https://api.github.com')
-    .get('/repos/tianhaoz95/approveman-test/pulls/1/reviews')
-    .reply(200, reviews)
-}
+export const setPreviousReviews = (reviews: any[]): void => {
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)